src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -395,8 +395,11 @@ private void populateLocalReportSummaryDetail(ListingData listingData, List<Subm
                         }
                         var adhocReportType = new AdhocReportType();
                         adhocReportType.setDate(dateListedTypedItem.getValue().getListedDate());
-                        adhocReportType.setMultSub(
-                                caseData.getMultipleReference() + ", " + caseData.getSubMultipleName());
+                        var mulRef = Strings.isNullOrEmpty(caseData.getMultipleReference())
```

Comment:
instead of doing this we can use the StringUtils.defaultString() method from apache.commons package. e.g. `StringUtils.defaultString(caseData.getMultipleReference(), "")`

Reply:
Why is this being done? Won't this simply change the value of multSub from "null, null" to ", "? Why is the new value anymore valid than the old?